### PR TITLE
Remove duplicate information on citing

### PR DESCRIPTION
**Pull request by BenjaminRodenberg**
Follow-up to https://github.com/precice/precice.github.io/pull/97

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,15 +73,9 @@ Please refer to [our website](https://www.precice.org/adapter-fenics.html#how-ca
 
 To create and install the `fenicsprecice` python package the following instructions were used: [How To Package Your Python Code from python-packaging.readthedocs.io](https://python-packaging.readthedocs.io/en/latest/index.html).
 
-## Citing
+## References & Citing
```
Comment (ajaust) [inline README.md:76]: "References" has been added, but there is no reference to "references" in the text. I will make a suggestion for the text below.

Maybe we want to chose a slightly different name for this section?
```suggestion
## How to cite and further literature
```

File: README.md [hunk 2]
```diff
@@ -73,15 +73,9 @@ Please refer to [our website](https://www.precice.org/adapter-fenics.html#how-ca
...
 
-If you are using this adapter, please refer to the [citing information on the FEniCS adapter](https://www.precice.org/adapter-fenics.html#how-to-cite).
-
-preCICE is an academic project, developed at the [Technical University of Munich](https://www5.in.tum.de/) and at the [University of Stuttgart](https://www.ipvs.uni-stuttgart.de/). If you use preCICE, please [cite us](https://www.precice.org/publications/):
-
-*H.-J. Bungartz, F. Lindner, B. Gatzhammer, M. Mehl, K. Scheufele, A. Shukaev, and B. Uekermann: preCICE - A Fully Parallel Library for Multi-Physics Surface Coupling. Computers and Fluids, 141, 250–258, 2016.*
-
-If you are using FEniCS, please also consider the information on [the official FEniCS website on citing](https://fenicsproject.org/citing/).
+If you are using this FEniCS adapter, please refer to the [citing information on the FEniCS adapter](https://precice.org/adapter-fenics.html#how-to-cite) and also consider the information on [the official FEniCS website on citing](https://fenicsproject.org/citing/).
```
Comment (ajaust) [inline README.md:78]:
```suggestion
If you are using this FEniCS adapter and want to cite it, please refer to the [citing information on the FEniCS adapter](https://precice.org/adapter-fenics.html#how-to-cite). You can find further literature related to the FEniCS adapter [in the adapter's documentation](https://precice.org/adapter-fenics.html#related-literature).

Citation information and further literature about FEniCS cac be found on the [official FEniCS website](https://fenicsproject.org/citing/).
```

Comment (ajaust) [inline README.md:78]:
```suggestion
If you are using this FEniCS adapter and want to cite it, please refer to the [citing information on the FEniCS adapter](https://precice.org/adapter-fenics.html#how-to-cite). You can find further literature related to the FEniCS adapter [in the adapter's documentation](https://precice.org/adapter-fenics.html#related-literature).

Citation information and further literature about FEniCS cac be found on the [official FEniCS website](https://fenicsproject.org/citing/).
```

**Review by ajaust — COMMENTED** [review]
I added some suggestions. I would like to split the citation section a bit more to separate the FEniCS adapter (this repository) and FEniCS (third party). Thus I split the text into two paragraphs.

File: README.md (hunk 2 above)
Comment (BenjaminRodenberg) [reply README.md]: I think this is a duplicate of the suggestion from above, isn't it?

Comment (ajaust) [reply README.md]: Seems like it. I do not know how I managed to to this. So please feel free to ignore this.

**Comment by IshaanDesai** [comment]
Is this still relevant after https://github.com/precice/fenics-adapter/commit/41f53758ac5b9692b39945ccfd01253bbdeabb9d?

**Comment by ajaust** [comment]
I think it can be closed. I am happy with the current situation and @BenjaminRodenberg was fine with it in a private discussion as well.

Sorry for not informing @IshaanDesai and for not pushing to this PR nor opening a new PR. I pushed directly to develop by accident. I guess I am not used to have full access to this repository yet.

**Pull request closed without merging**